scripts/schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
`config/schema.go` is generated based on this, so I'm not really sure that counts as having the same text twice. It's not strictly necessary, but I think it's good practice to have the actual schema file around.